plugins/aws/acm/acmSingleDomainNameCertificate.js

```diff
@@ -0,0 +1,67 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'ACM Single Domain Name Certificates',
+    category: 'ACM',
+    domain: 'Identity and Access management',
+    description: 'Ensure that ACM single domain name certificates are used instead of wildcard certificates within your AWS account.',
```

Comment:
```suggestion
    description: 'Ensures that ACM single domain name certificates are used instead of wildcard certificates within your AWS account.',
```